### PR TITLE
Cardano Tiddies

### PR DESCRIPTION
**Pull request by yehboigapit**
Hi please verify my project Cardano Tiddies

### CODE REVIEW

**Comment by RedArchfiend** [comment]
Please provide link to social/website that shows the Policy ID.

**Comment by yehboigapit** [comment]
> Please provide link to social/website that shows the Policy ID.

hi here is my link. 😄 
https://cnft.io/token.php?id=615c23057e72620eb522d0f5

File: cardanotiddies [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "project": "Cardano Tiddies",
+    "tags": [
+        "titty"
+        "tiddy"
+        "cardano"
+        "ADA
+        
+    ],
+    "policies": [
+       69bdc2bff8f9d1f6a4e6b990e9c9845286c8461722850f6ac24c14bb
```
Comment (RedArchfiend) [inline cardanotiddies:11]: Add the speech marks at the beginning and the end of the Policy ID in the code.

Comment (yehboigapit) [reply cardanotiddies]: > Add the speech marks at the beginning and the end of the Policy ID in the code.

Hi. was able to edit. hope it reflects on your end

**Pull request merged**